editor/debugger/script_editor_debugger.cpp

```diff
@@ -1910,6 +2004,10 @@ void ScriptEditorDebugger::toggle_profiler(const String &p_profiler, bool p_enab
 }
 
 ScriptEditorDebugger::ScriptEditorDebugger() {
+	if (parse_message_handlers == nullptr) {
```

Comment:
Check can be `unlikely(parse_message_handlers.is_empty())`, if it was the reason to use pointer.

Reply:
Changed – thanks @bruvzg!